FindPoetry.cmake

```diff
@@ -19,7 +19,7 @@ else()
 endif()
 
 
-set(LOCALINSTALL_POETRY_DIR "${CMAKE_SOURCE_DIR}/poetry")
+set(LOCALINSTALL_POETRY_DIR "${CMAKE_SOURCE_DIR}/stm32-tools/poetry")
 message(STATUS "Downloading poetry install script to: ${LOCALINSTALL_POETRY_DIR}")
 
 message(STATUS "Installing Poetry")
```

Comment:
Missed this in the last PR - the `PREFIX` in `FetchContent_Declare` should actually just be `${CMAKE_SOURCE_DIR}/stm32-tools/`, and then the SOURCE_DIR stays the same.

Reply:
I think your suggestion and my current implementation are incorrect.

What I get with your suggestion is:
```
stm32-tools 
  poetry
    src
    tmp
    install_poetry.py
  src
  tmp
```

What I get with my current implementation is:
```
stm32-tools 
  poetry
    src
      install_poetry.py
    tmp
    install_poetry.py
```

What I want is:
```
stm32-tools
  poetry
    install_poetry.py 
```    

What I actually need to do is remove `PREFIX`